Cargo.toml

```diff
@@ -11,15 +11,16 @@ edition = "2021"
 rust-version = "1.60"
 
 [features]
-default = ["std", "multihash/default"]
-std = ["multihash/std", "unsigned-varint/std", "alloc", "multibase/std", "serde/std"]
+default = ["std", "multihash/default", "multihash-codetable/default"]
+std = ["multihash/std", "multihash-codetable/std", "unsigned-varint/std", "alloc", "multibase/std", "serde/std"]
 alloc = ["multibase", "multihash/alloc", "core2/alloc", "serde/alloc"]
-arb = ["quickcheck", "rand", "multihash/arb", "multihash/multihash-impl", "multihash/sha2", "arbitrary"]
+arb = ["quickcheck", "rand", "multihash/arb", "arbitrary"]
 scale-codec = ["parity-scale-codec", "multihash/scale-codec"]
 serde-codec = ["alloc", "serde", "multihash/serde-codec", "serde_bytes"]
 
 [dependencies]
-multihash = { version = "0.18.0", default-features = false }
+multihash = { version = "0.19.0", default-features = false }
+multihash-codetable = { version = "0.1.0", default-features = false, features = ["digest", "sha2"] }
```

Comment:
With the upgrade to of Multihash, we should take the chance of reducing the dependency tree. From this PR, it looks like `multihash-codetable` is only used in the tests/examples. Therefore I propose making it a dev dependency and not re-exporting it.

I know it's a breaking change, but the code table was only re-exported as the full library was. With separate crates we now have more flexibility here.

Reply:
Happy to :)